test/Microsoft.AspNet.Razor.Test/CodeGeneration/AttributeBlockChunkGeneratorTest.cs

```diff
@@ -4,36 +4,36 @@
 using Microsoft.AspNet.Razor.Text;
 using Xunit;
 
-namespace Microsoft.AspNet.Razor.Generator
+namespace Microsoft.AspNet.Razor.Chunks.Generators
```

Comment:
Could you also move the type to `/test/Microsoft.AspNet.Razor.Test/Chunks/Generators` directory?